### ISSUE TITLE
Mist cranks away "creating wallet" for a day -still no wallet

### ISSUE THREAD

**Issue by neonjeff** [ethereum/mist#2878]
``````
Version: `0.9.0`
OS & Version: windows
Node type: `geth`
Number of blocks synchronized: ? 4,110,0221 blocks 11 user sync
``````
Mist and Ethereum Wallet running on windows 10 8 core cpu AMD radeon r9
24G RAM
current ETH block sync looks fine with Mist
selected single signature wallet with a little over $20 bucks in account
it removed about $6 and started creating wallet
its been cranking along for a day - 24 hours!
How long is it supposed to take?
Any advise?

**Comment by evertonfraga** [comment]
Hello, 

First, check if the transaction went through (using https://etherscan.io). If it didn't, you could try downloading the **Ethereum Wallet** (not mist) 0.8.10, as the latest version has a bug around contracts (fix being prepared).

Thanks for reporting, and sorry for the delay.

**Comment by evertonfraga** [comment]
This issue was fixed in the latest 0.9.3.
